src/mock/parsec/tests.rs

```diff
@@ -32,13 +32,11 @@ fn smoke() {
     let _ = genesis_group.insert(bob_id);
 
     let mut alice = from_genesis(alice_id, &genesis_group, ConsensusMode::Supermajority);
-
-    let mut bob = from_genesis(bob_id, &genesis_group, ConsensusMode::Supermajority);
-
     alice
         .vote_for(Observation::OpaquePayload(Payload(1)))
         .unwrap();
 
+    let mut bob = from_genesis(bob_id, &genesis_group, ConsensusMode::Supermajority);
```

Comment:
just wondering, whether this change of execution order is because of any refactoring work?

Reply:
This change of execution order is so that the test if failing if the fix is not there.